examples/webvr_cubes.html

```diff
@@ -127,7 +127,7 @@
 
```

Comment:
@toji I added `preserveDrawingBuffer: true` to the WebGLRenderer options (per your example: https://github.com/toji/webvr-samples/blob/gh-pages/04-simple-mirroring.html), it fixed the judder that I experienced on DK2... I guess the `preserveDrawingBuffer` property should be set according to the `hasExternalDisplay` capability in all of the webvr examples?